quartz/components/Footer.tsx

```diff
@@ -12,7 +13,7 @@ export default ((opts?: Options) => {
     const year = new Date().getFullYear()
     const links = opts?.links ?? []
     return (
-      <footer class={`${displayClass ?? ""}`}>
+      <footer class={classNames(displayClass)}>
```

Comment:
let's separate this out to a standalone PR?

Reply:
Will do.